src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -229,25 +212,41 @@ public ResponseEntity<Void> cutAndPasteNode(@PathVariable("studyUuid") UUID stud
         return ResponseEntity.ok().build();
     }
 
-    @GetMapping(value = "/studies/{studyUuid}/network")
+    @RequestMapping(value = "/studies/{studyUuid}/network", method = RequestMethod.HEAD)
     @Operation(summary = "check study root network existence")
     @ApiResponses(value = {
-        @ApiResponse(responseCode = "204", description = "The network does exists"),
-        @ApiResponse(responseCode = "404", description = "The network doesn't exist - Will try to reimport the network if params is set to true")})
-    public ResponseEntity<Network> getStudyNetwork(@PathVariable("studyUuid") UUID studyUuid,
-                                                   @RequestParam("reimportNetworkIfNotFound") boolean reimportNetworkIfNotFound,
-                                                   @RequestHeader(HEADER_USER_ID) String userId) {
+        @ApiResponse(responseCode = "200", description = "The network does exists"),
+        @ApiResponse(responseCode = "204", description = "The network doesn't exist")})
+    public ResponseEntity<Void> checkNetworkExistence(@PathVariable("studyUuid") UUID studyUuid) {
         UUID networkUUID = networkStoreService.getNetworkUuid(studyUuid);
-        try {
-            networkStoreService.getNetwork(networkUUID, PreloadingStrategy.NONE, null);
+        return networkStoreService.isNetworkExisting(networkUUID)
+            ? ResponseEntity.ok().build()
+            : ResponseEntity.noContent().build();
 
-            return ResponseEntity.noContent().build();
-        } catch (ResponseStatusException e) {
-            if (reimportNetworkIfNotFound) {
-                studyService.reimportStudyNetwork(studyUuid, userId);
-            }
-            throw e;
-        }
+    }
+
+    @PostMapping(value = "/studies/{studyUuid}/network", params = {"caseUuid"})
+    @Operation(summary = "recreate study network of a study from an existing case")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "Study network import has started"),
+        @ApiResponse(responseCode = "424", description = "The case doesn't exist")})
```

Comment:
```suggestion
        @ApiResponse(responseCode = "424", description = "The study case doesn't exist")})
```
Like below ?

Reply:
Ok, then the correct sentence below is "the study's case"

Reply:
fixed